src/transaction/log_replication_atomic.cpp

```diff
@@ -45,6 +45,8 @@ namespace cublog
      * and, thus, does not need to be left in consistent state; thus, no check as to the consistent
      * termination state for atomic replication is needed
      */
+
+    cleanup_lock_resources ();
```

Comment:
It should be called before calling lock_finalize(). When I check it, it is already called like that. However, add comment to leave information for others. The method of leaving comments mimics existing code, and it would be nice if there was a clearer way.
```
xboot_shutdown_server()
{
    ...
    pts_ptr->finish_replication_during_shutdown (*thread_p); // stop receiving log before log_final()
```